gpep517/__main__.py

```diff
@@ -12,6 +12,7 @@
 import tempfile
 
 from pathlib import Path
+from typing import Optional
```

Comment:
I'm sorry but elsewhere in the code I import the whole `typing` module, so please keep the style consistent.

Reply:
Fixed.